backup-loop.sh

```diff
@@ -206,28 +206,53 @@ tar() {
   }
   backup() {
     ts=$(date +"%Y%m%d-%H%M%S")
-    outFile="${DEST_DIR}/${BACKUP_NAME}-${ts}.${backup_extension}"
-    tries=3
-    while (( tries-- > 0 )); do
-      log INFO "Backing up content in ${SRC_DIR} to ${outFile}"
-      command tar "${excludes[@]}" "${tar_parameters[@]}" -cf "${outFile}" -C "${SRC_DIR}" . || exitCode=$?
-      if [ ${exitCode:-0} -eq 0 ]; then
-        break
-      elif [ ${exitCode:-0} -eq 1 ]; then
-        if (( tries > 0 )); then
-          log INFO "...retrying backup in 5 seconds"
-          sleep 5
-          continue
-        else
-          log WARN "Giving up on this round of backup"
-        fi
+    outFile="${DEST_DIR}/${BACKUP_NAME}-${ts}.tar"
+    log INFO "Backing up content in ${SRC_DIR} to ${outFile}"
+
+    # backup dat and dat_old files separately
+    (cd "${SRC_DIR}" && find . -type f -name "*.dat" -o -name "*.dat_old" ) | command tar "${excludes[@]}" -cf "${outFile}" -C "${SRC_DIR}" -T - || exitCode=$?
```

Comment:
Doesn't hurt other than size, but do the dat_old's need to be backed up?

Reply:
Better to backup them for consistency with old behavior.